pym/bob/utils.py

```diff
@@ -641,7 +641,7 @@ def __init__(self):
             origSigInt = signal.getsignal(signal.SIGINT)
             signal.signal(signal.SIGINT, signal.SIG_IGN)
 
-            method = 'fork' if sys.platform == 'msys' else 'forkserver'
+            method = 'fork' if isMsys else 'forkserver'
```

Comment:
`isWindows()`?

Reply:
i have no idea, currently for win32 and linux we used 'forkserver' and just for msys we used 'fork' ?!
if i use `isWindows()` now, i will change the behavior.

Reply:
No. The `win32` case is already tested above. This leaves us either with a POSIX emulation on Windows (MSYS2 or Cygwin) or a native POSIX system. This was never tested on Cygwin but it should also use the `fork` method.